installer/inventory

```diff
@@ -26,14 +26,18 @@ awx_secret_key=awxsecret
 # openshift_user=developer
 # awx_node_port=30083
 
-# Standalone Docker Install
+# Docker Compose Install
+use_docker_compose=false
+# The docker_compose.yml file will be created in this directory
+# The name of the directory (here "awx") will be the prefix of the docker containers
+docker_compose_dir=/var/lib/awx
+
+# Common Docker parameters
 postgres_data_dir=/tmp/pgdocker
 host_port=80
 
-# Required for Openshift when building the image on your own
-# Optional for Openshift if using Dockerhub or another prebuilt registry
-# Required for Standalone Docker Install if building the image on your own
-# Optional for Standalone Docker Install if using Dockerhub or another prebuilt registry
+# Required when building the image on your own (with any method)
+# Optional if using Dockerhub or another prebuilt registry (with any method)
```

Comment:
Leave the openshift notes here.

Reply:
Ok, but why ? we're repeating the same thing twice...?